addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -125,3 +125,11 @@ def test_region_SN_monkey_patch(self):
             self.PhoneInputOutputLine("+22176 707 0065"),
         )
         self._assert_parsing_phonenumbers(parse_test_lines_SN)
+
+    def test_region_CO_monkey_patch(self):
```

Comment:
alphabetical ordering :D (reason; tests are executed alphabetically in a given class so better define them the same way, easier when debugging)

Reply:
Deno ! (It's "Done" but alphabetically ordered)